storage/mysql/tree_storage.go

```diff
@@ -36,7 +36,7 @@ const (
 	insertSubtreeMultiSQL = `INSERT INTO Subtree(TreeId, SubtreeId, Nodes, SubtreeRevision) ` + placeholderSQL
 	insertTreeHeadSQL     = `INSERT INTO TreeHead(TreeId,TreeHeadTimestamp,TreeSize,RootHash,TreeRevision,RootSignature)
 		 VALUES(?,?,?,?,?,?)`
-	selectNonDeletedTreeIDByTypeAndStateSQL = "SELECT TreeId FROM Trees WHERE TreeType = ? AND TreeState = ? AND (Deleted IS NULL OR Deleted = 'false')"
+	selectNonDeletedTreeIDByTypeAndStateSQL = "SELECT TreeId FROM Trees WHERE TreeType IN (" + placeholderSQL + ") AND TreeState = ? AND (Deleted IS NULL OR Deleted = 'false')"
```

Comment:
@Martin2112 Since this query is used only with fixed number of parameters (2), do you think we should just write "IN (?, ?)" instead of `placeholderSQL`? Similarly to [how]( https://github.com/google/trillian/pull/970/files#diff-fda46990892de1b2549108efd05f30e2R45) you did in another RP with `TreeState`.

Reply:
Did it.